### PR TITLE
Update README with libfabric dependencies

### PR DESCRIPTION
**Pull request by Sakoram**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -60,8 +60,28 @@ Detailed information about MCM SDK can be found in [sdk](sdk) directory.
 ### Basic Installation
 
 0. **Install Dependencies**
-    - MTL: Follow the [MTL setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/build.md) for installation.
-    - gRPC: Refer to the [gRPC documentation](https://grpc.io/docs/languages/cpp/quickstart/) for installation instructions.
+
```
Comment (ko80) [inline docs/README.md:63]: Zero numbering looks weird
image

Comment (ko80) [reply docs/README.md:63]: 1. Install dependencies
    - MTL
    - gRPC
    - libbsd
        - Ubuntu/Debian
        - Centos
     - libfabric

2. Clone the repo

Comment (Sakoram) [reply docs/README.md]: done

File: docs/README.md [hunk 2]
```diff
@@ -59,26 +59,43 @@ Detailed information about MCM SDK can be found in [sdk](sdk) directory.
...
+        - Ubuntu/Debian
+        ```bash
+        sudo apt-get update
+        sudo apt-get install libbsd-dev
+        ```
+        - Centos stream
+        ```bash
+        sudo yum install -y libbsd-devel
+        ```
+    - Install Libfabric. Follow [libfabric documentation](https://github.com/ofiwg/libfabric#readme-ov-file) or simply use following commands
```
Comment (ko80) [inline docs/README.md:76]:
```suggestion
    - Install Libfabric. Follow [libfabric documentation](https://github.com/ofiwg/libfabric#readme-ov-file) or simply use the following commands
```

**Review by ko80 — APPROVED** [review]
Looks good

**Pull request merged**